### ISSUE TITLE
Color theme

### ISSUE THREAD

**Issue by rasmuse** [demokratikollen/demokratikollen#83]
We need to define a common set of colors in matching tones that can be used throughout the site. Most colors already carry meaning in terms of parties, so this requires some serious thought.

My suggestion to start with is to make 1) a neutral greyish set of colors, ranging from almost-black to almost-white and 2) a slightly more brownish set of colors that don't resemble any of the party colors, which can be used when a lot of different tones are needed, e.g. in pie charts with the ~20 different committees.

I'll get right to it.

**Comment by rasmuse** [comment]
OK, just pushed a few style changes to `develop` (most important changes in c198a1f21f46cb586dc3ff3ec161058b12c3e103).

Most importantly, white is not white. The `#fff` background is exchanged for a very pale grey. In Sass, I defined two arrays of colors: `$greyish-0`, `$greyish-1`, ..., `$greyish-9` and `$brownish-0` through `$brownish-9`. You don't have to worry too much about it yet, but at least I will try to work with these colors for a while and see how it feels, and incrementally make changes and additions.

Also I started on some common styling for charts, including `d3-tip`. See `www/design/sass/_charts.scss`.
